src/Sulu/Bundle/ContactBundle/Contact/CustomerManager.php

```diff
@@ -0,0 +1,109 @@
+<?php
+/*
+ * This file is part of Sulu.
+ *
+ * (c) MASSIVE ART WebServices GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Sulu\Bundle\ContactBundle\Contact;
+
+use Doctrine\ORM\EntityManager;
+use Sulu\Bundle\ContactBundle\Util\IdsHandlingTrait;
+
+/**
+ * Implements functionality for the manager of account and contact combination.
+ */
+class CustomerManager implements CustomerManagerInterface
+{
+    use IdsHandlingTrait;
+
+    /**
+     * @var EntityManager
+     */
+    private $entityManager;
+
+    /**
+     * @var string
+     */
+    private $contactEntityClass;
+
+    /**
+     * @var string
+     */
+    private $accountEntityClass;
+
+    public function __construct(EntityManager $entityManager, $contactEntityClass, $accountEntityClass)
+    {
+        $this->entityManager = $entityManager;
+        $this->contactEntityClass = $contactEntityClass;
+        $this->accountEntityClass = $accountEntityClass;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function findByIds($ids)
+    {
+        if (count($ids) === 0) {
+            return [];
+        }
+
+        $parsed = $this->parseIds($ids, ['a' => [], 'c' => []]);
+
+        $accounts = $this->findAccountsByIds($parsed['a']);
+        $contacts = $this->findContactsByIds($parsed['c']);
```

Comment:
Isn't it possible to use the other services here? Or do you need a special representation?